frontend/controller/actions/identity-kv.js

```diff
@@ -251,7 +251,7 @@ export default (sbp('sbp/selectors/register', {
   'gi.actions/identity/kv/loadNotificationStatus': () => {
     return sbp('okTurtles.eventQueue/queueEvent', KV_QUEUE, async () => {
       const status = await sbp('gi.actions/identity/kv/fetchNotificationStatus')
-      sbp('state/vuex/commit', 'setNotificationStatus', status)
+      sbp('gi.notifications/setNotificationStatus', status)
```

Comment:
This no longer seems to perform any modifications to the Vuex state. Is that intentional?

Reply:
Yes, that's very much intentional, since there's no Vuex in the SW. However, this will result in an event that ultimately sets the Vuex state.

Reply:
See this addition: `sbp('okTurtles.events/emit', NOTIFICATION_STATUS_LOADED, status)`